internal/v1/handler_blueprints.go

```diff
@@ -71,3 +72,38 @@ func (h *Handlers) CreateBlueprint(ctx echo.Context) error {
 		Id: id,
 	})
 }
+
+func (h *Handlers) ComposeBlueprint(ctx echo.Context, id openapi_types.UUID) error {
+	idHeader, err := getIdentityHeader(ctx)
+	if err != nil {
+		return err
+	}
+
+	blueprintEntry, err := h.server.db.GetBlueprint(id, idHeader.Identity.OrgID, idHeader.Identity.AccountNumber)
+	if err != nil {
+		return err
+	}
+	blueprint, err := BlueprintFromEntry(blueprintEntry)
+	if err != nil {
+		return err
+	}
+	composeResponses := make([]ComposeResponse, 0, len(blueprint.ImageRequests))
+	clientId := ClientId("api")
+	for _, imageRequest := range blueprint.ImageRequests {
+		composeRequest := ComposeRequest{
+			Customizations:   &blueprint.Customizations,
+			Distribution:     blueprint.Distribution,
+			ImageRequests:    []ImageRequest{imageRequest},
+			ImageName:        &blueprint.Name,
+			ImageDescription: &blueprint.Description,
+			ClientId:         &clientId,
+		}
+		composesResponse, err := h.handleCommonCompose(ctx, composeRequest, blueprintEntry.Id)
```

Comment:
We're passing the blueprint id here, should we pass version id instead?

Also adding a test for this handler could help.

Reply:
indeed,  but we don't have the versioning implemented, right?

Reply:
I'm not sure what you mean, but having corrupted data is definitelly wrong.
We do have versioning implemented to the level of having the first version of blueprint inserted.
We can easily adjust the BlueprintEntry to fetch that id as well.

Reply:
I though that we want for wait a  bit until the versioning/editing will be implemented and than do a rebase, I didn't notice that the DB is already implemented. 

I added a `VersionId` field to `BlueprintEntry`, hopefully it answers this concern.

Reply:
Yup, that's perfect! :)